sdks/java/io/cdc/src/main/java/org/apache/beam/io/cdc/KafkaSourceConsumerFn.java

```diff
@@ -42,9 +43,11 @@
   private static final Logger LOG = LoggerFactory.getLogger(KafkaSourceConsumerFn.class);
   public static final String BEAM_INSTANCE_PROPERTY = "beam.parent.instance";
 
+  public static long minutesToRun = 2; // Pass this as parameter ??
```

Comment:
Can we make this configurable? User should be able to provide this value as parameter (optional). If parameter is not provided then we can ignore and run without stoping it.